src/react-apps/applications/dashboard/__tests__/components/base/knowIssues.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { mount } from 'enzyme';
+import 'jest';
+import * as React from 'react';
+import * as networking from '../../../../shared/src/utils/networking';
+import { KnownIssuesComponent } from '../../../src/features/knowIssues/knownIssues';
+
+describe('>>> components/base/knownIssues.tsx', () => {
```

Comment:
Do you have any tests that asserts that you show any text retreived from "docs"?